api.py

```diff
@@ -1,3 +1,118 @@
+from models.change import Change
+from models.consultation import Consultation
+import json
+
+import requests
+
 class API:
```

Comment:
All, what u can commend in this file is:
* This class sends GET-requests to tthkAPI.

And this all as well.